package.json

```diff
@@ -14,11 +14,7 @@
         "lint:eslint": "eslint '*.{js,jsx,ts,tsx}' --cache --format stylish",
         "fix": "yarn fix:eslint && yarn fix:prettier",
         "fix:prettier": "prettier --write '*.{js,jsx,ts,tsx}'",
-        "fix:eslint": "eslint '*.{js,jsx,ts,tsx}' --format stylish --fix",
-        "test": "jest -c test/jest.config.js",
```

Comment:
Remember to also remove `jest` from dependencies!

Reply:
I believe nock is for http server mocking so I believe that can be removed too. https://www.npmjs.com/package/nock